packages/schematics/angular/e2e/files/src/app.e2e-spec.ts.template

```diff
@@ -8,9 +8,9 @@ describe('workspace-project App', () => {
     page = new AppPage();
   });
 
-  it('should display welcome message', () => {
-    page.navigateTo();
-    expect(page.getTitleText()).toEqual('<%= relatedAppName %> app is running!');
+  it('should display welcome message', async () => {
```

Comment:
Protractor uses a custom control flow mechanism that handles the promises.  These are technically not floating.  This is something that will eventually be addressed but needs to be coordinated with protractor and migrations need to be put in place to synchronous behavior of existing projects with new projects.
There's an open PR with some more details here: https://github.com/angular/angular-cli/pull/15472

Reply:
Okay, I'll leave these as is by removing them from my commits.